core/types/transaction.go

```diff
@@ -51,6 +51,8 @@ const (
 	DynamicFeeTxType = 0x02
 	BlobTxType       = 0x03
 	Rip7560Type      = 0x04
+	// TODO: do we really need a new type?
+	Rip7560BundleHeaderType = 0x05
```

Comment:
I think we'd better discuss this w/ 4337 team about the intention of the spec.

Reply:
Yeah, that's a good idea